inscriptions/ISic000709.xml

```diff
@@ -190,11 +195,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AL">
+                <ab>
+                    <lb n="1"/><persName><name type="praenomen"><expan><abbr>P</abbr><ex>ublius</ex></expan></name> <name type="nomen">Scrib<unclear>o</unclear>
+                    <lb n="2"/>nius</name> <name type="cognomen">Sol<unclear>le</unclear>mnis</name></persName> 
```

Comment:
`<lb n="2" break="no"/>` is used to encode the fact that the word at the end of line 1 continues onto line 2 without a word-break.